### ISSUE TITLE
linter: importing it/test from elsewhere than vitest reports `no-standalone-expect`

### ISSUE THREAD

**Issue by Ericlm** [oxc-project/oxc#22826]
### What version of Oxlint are you using?

1.61.0

### What command did you run?

oxlint

### What does your `.oxlintrc.json` (or `oxlint.config.ts`) config file look like?

```jsonc
{
  "$schema": "./node_modules/oxlint/configuration_schema.json",
  "plugins": ["eslint", "typescript", "unicorn", "oxc", "vue", "vitest"],
  "env": {
    "browser": true
  },
  "categories": {
    "correctness": "error"
  }
}
```

### What happened?

**Reproduction:** https://github.com/Ericlm/oxlint-vitest

- Create a setup file that extends "it" or "test" from vitest
- In a test file, create a simple assertion while using the customized it/test
- The error `eslint-plugin-jest(no-standalone-expect)` is reported
- No problem when using the import from `vitest`

Issue : 
```ts
import { expect } from 'vitest'
import { it } from './vitest.setup.ts'

it('renders name', async () => {
  expect(true).toBe(true)
})
```

No issue : 
```ts
import { expect, it } from 'vitest'

it('renders name', async () => {
  expect(true).toBe(true)
})
```

**Comment by kapobajza** [comment]
Hello. Seems like this is also the case with the upstream `eslint-plugin-vitest` ESLint plugin. Here's a link to the playground:

https://eslint-online-playground.netlify.app/#eNp1U8tOwzAQ/BXLl7RV69yDQAj4C8zBJJvKJVkHPyqqKv/O+tGWQvEhtrXj3ZnR5MidbWv4UuM0gPDgvNg53nA9TsZ6dmTwNUHr2cx6a0Ym+V5HkOQSzxD9oyzqDBAOfJiEdxFJWL+Q3AJ2YB1DNYLka6bcAVu2WLL7B3aUyMqshbcBlsKbJ8hHifPyTiJfc3CDRi9ag73e/iLaQa8RnlPpwie/qPMLyalNebBzJ8hjweyI66WeZWRM9ZhvdUZupiFsNVZRGFGOaBquwpD2M4lFlPQaP4ymFdJOWGjNOEYrunUuJu1x2TCAay53xj6DocENteFgrbHRNsk7E94HsvCtNIhLcjSbgMFBt9krS1oaVpm+r86YuZxO+3lMr9PY12q1qlerUwiqS/es95pZduRP81sihBAlFDc8EAl/JaSYTXqcV9ipwSBscjaSquQEuV8mF1Vxe5NYkjKp9kNtgXQYpJQkNpL7w0S2NXQaTUdzyc5c6GD/AlMkhK2GZF4RcJ2P+HRQ+Q8onE8Z+6d4MzrX2Agl+jOfvwHu/idT

**Comment by kapobajza** [comment]
Additionally: one way to get this working is to rename your function to something other than `it`, and to also add it to the [`additionalTestBlockFunctions`](https://oxc.rs/docs/guide/usage/linter/rules/jest/no-standalone-expect.html#additionaltestblockfunctions) config option. Here's an example:

```json
{
  "$schema": "./node_modules/oxlint/configuration_schema.json",
  "plugins": ["eslint", "typescript", "unicorn", "oxc", "vue", "vitest"],
  "env": {
    "browser": true
  },
  "categories": {
    "correctness": "error"
  },
  "rules": {
    "vitest/no-standalone-expect": [
      "error",
      {
        "additionalTestBlockFunctions": ["myIt"]
      }
    ]
  }
}
```

And then you can do this:

```ts
import { expect } from 'vitest'
import { myIt } from './vitest.setup.ts'

myIt('renders name', async () => {
  expect(true).toBe(true)
})
```

Now the error won't be reported.

**Comment by camc314** [comment]
Hi! thanks for reporting this issue.

@ kapobajza has summarised it quite nicely. but to add>

Addding the following:
```
      {
        "additionalTestBlockFunctions": ["it"]
      }
```

should also allow the rule to pick up and flag these errors.

This behaviouir is aligned with vitest (iirc)

Thanks!

**Comment by Ericlm** [comment]
Thank you for your answers!
